fe/fe-core/src/main/java/org/apache/doris/cloud/datasource/CloudInternalCatalog.java

```diff
@@ -701,6 +701,32 @@ public void dropMaterializedIndex(long tableId, List<Long> indexIds, boolean dro
         }
     }
 
+    /**
+     * for cloud mode, drop rollupIndex/materializedIndexId in kv meta store
+     * @param tableId
+     * @param materializedIndexId
+     */
+    public void eraseIndexDropBackendReplicas(long tableId, List<Long> indexIdList) {
```

Comment:
This is a cofusing name, consider `eraseDroppedIndex`

Reply:
> This is a cofusing name, consider `eraseDroppedIndex`

resolved